src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -39,23 +40,29 @@
 @Api(tags = "Actors")
 public class ActorsController {
 
-    @Autowired
-    BuildConfig buildConfig;
-
     @Autowired
     private ActorsRepository actorRepository;
 
     @RequestMapping(value = "", method = RequestMethod.GET)
     @ApiOperation(value = "Get all actors", notes = "Retrieve and return all actors")
     @ApiResponses(value = { @ApiResponse(code = 200, message = "List of actor objects") })    
-public Flux<Actor> finActors(ServerHttpResponse response) {
+    public Mono<Actor> findByActorId(ServerHttpResponse response) {
+        return actorRepository.findByActorId("nm0000502");
+    }
```

Comment:
return actorRepository.findByActorId("nm0000502") <-- needs param instead of hardcoded value